### ISSUE TITLE
Blocked to login screen after login

### ISSUE THREAD

**Issue by Wip-IT-Tech-eCom** [akeneo/pim-community-dev#15394]
:bug: **I'm reporting a Bug** :bug:

I updated my PIM from 5.0.37 to the latest 5.0.48 Version using as usual this tutorial:

**Comment by navneetbhardwaj** [comment]
Hi Wip-IT-Tech-eCom,

Share the prod.log and browser console screen error to check what exactly the issue.

**Comment by Wip-IT-Tech-eCom** [comment]
Hello @navneetbhardwaj ,

I remarked theses errors at the end of my dev.log (I use Akeneo in Dev mode):

`[2021-10-03 11:03:32] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/tabulator_bootstrap4.min.css.map"" at /home/pimwip/pim/vendor/symfony/http-kernel/Ev$`

`[2021-10-03 11:03:36] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /js/extensions.json" (from "http://akeneo-pim.wip.local/")" at /home/pimwip/pim/vendor/sy$`

I also noticed this in the browser console screen:

`XHR GET http://akeneo-pim.wip.local/js/extensions.json?version=mzloum
[HTTP/1.1 404 Not Found 528ms]`

`Uncaught Error: It seems that your web server is not well
                     configured as we were not able to load the frontend
                      configuration. The most likely reason is that the
                       mod_rewrite module is not installed/enabled.
    n http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    o http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    fireWith http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    f http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    setTimeout handler*i/< http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    o http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    fireWith http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    fire http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    o http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    fireWith http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    a[0] http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    o http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    fireWith http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    C http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    t http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    send http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    ajax http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    t http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    o http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    getExtensionMap http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    getFormMeta http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    o http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    build http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    o http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    fRQL http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    a http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    0 http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    a http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    i http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    <anonymous> http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    <anonymous> http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
main.min.js:1:1915577
    n http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    o http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    fireWith http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    f http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    (Async: setTimeout handler)
    i http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    o http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    fireWith http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    fire http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    o http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    fireWith http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    a[0] http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    o http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    fireWith http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    C http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    t http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    (Async: EventHandlerNonNull)
    send http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    ajax http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    t http://akeneo-pim.wip.local/dist/vendor.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    o http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    getExtensionMap http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    getFormMeta http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    o http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    build http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    o http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    fRQL http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    a http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    0 http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    a http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    i http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    <anonym> http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
    <anonym> http://akeneo-pim.wip.local/dist/main.min.js?ce26972fae60b42aab3e55f6d0aed186:1
`
`Error: Promised response from onMessage listener went out of scope background.js:841:170`

Best,
Victor

**Comment by Wip-IT-Tech-eCom** [comment]
Update:
I searched on found this topic: https://github.com/akeneo/pim-community-dev/issues/13575

I ran the command  `yarn update-extensions` after I did again the [patch update process](https://docs.akeneo.com/5.0/migrate_pim/apply_patch/apply_patch_ce.html)

I don't see anymore this time the error after the login but I'm still blocked and get an **503 error** when I want to go to the different system sections of my PIM:
![grafik](https://user-images.githubusercontent.com/48508597/135757698-864552db-d62b-4512-b22e-327a09f937e2.png)

![grafik](https://user-images.githubusercontent.com/48508597/135757740-9cf39257-7979-4057-b185-2bd8417a4a13.png)

I am also blocked on the loading screen on the Products section:

![grafik](https://user-images.githubusercontent.com/48508597/135758927-08de7895-728e-4608-ba26-a0429dbc19ac.png)


![grafik](https://user-images.githubusercontent.com/48508597/135758020-0136bdb4-47c0-4b88-9019-0b232e7aa0b3.png)

I noticed as well this error into dev.log:

`[2021-10-03 16:13:47] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "G$`

**Comment by Wip-IT-Tech-eCom** [comment]
Hi,

I noticed as well this error:
![grafik](https://user-images.githubusercontent.com/48508597/135871759-0d30accb-c8c6-4ae8-b95b-187c3a104b03.png)

![grafik](https://user-images.githubusercontent.com/48508597/135871810-73a192fa-3cc2-42ba-8c7b-d168c5bb1db8.png)

``[2021-10-04 16:06:39] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/tabulator_bootstrap4.min.css.map"" at /home/pimwip/pim/vendor/symfony/http-kernel/Ev$``

Best,
